sbt_common/internal_model/src/test/scala/uk/ac/wellcome/models/transformable/sierra/test/utils/SierraUtil.scala

```diff
@@ -0,0 +1,80 @@
+package uk.ac.wellcome.models.transformable.sierra.test.utils
+
+import java.time.Instant
+
+import uk.ac.wellcome.models.transformable.sierra.{
+  SierraBibRecord,
+  SierraItemRecord
+}
+
+import scala.collection.immutable.Stream
+import scala.util.Random
+
+trait SierraUtil {
+
+  // A lot of Sierra tests (e.g. mergers) check the behaviour when merging
+  // a record with a newer version, or vice versa.  Provide two dates here
+  // for convenience.
+  val olderDate: Instant = Instant.parse("1999-09-09T09:09:09Z")
+  val newerDate: Instant = Instant.parse("2001-01-01T01:01:01Z")
+
+  /** Returns a random digit as a string.  This is copied from the
+    * definition of Random.alphanumeric.
+    *
+    * @return
+    */
+  private def randomNumeric: Stream[Char] = {
+    def nextAlphaNum: Char = {
+      val chars = "0123456789"
```

Comment:
Is this nextAlphaNum or nextDigit?

perhaps `val chars = ('0' to '9')

Reply:
Copy-pasta error from `Random.alphaNumeric`, will fix